api/broker-client.yaml

```diff
@@ -1511,6 +1545,29 @@ components:
       type: array
       items:
         $ref: '#/components/schemas/Subscription'
+    Notification:
+      type: object
+      properties:
+        '@context':
+          $ref: '#/components/schemas/LdContext'
+        id:
+          type: string
+          format: uri
+        type:
+          $ref: '#/components/schemas/Name'
+        subscriptionId:
+          type: string
+          format: uri
+        notifiedAt:
+          type: string
+          format: date-time
+        data:
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/Entity'
+        trigger:
```

Comment:
i cannot find this field in the spec. Where is it coming from?

Reply:
Ken temporarily added this for my use case, because current implementation of orion-ld doesn't see a difference between entity creation and adding new attribute to existing entity. Once it'll be implemented, I'll remove it

Reply:
Do you know how this is planned to be done in the spec? You need to keep in mind that orion-ld is just one of the brokers that potentially can be used with the implementation. Some workaround from Ken is not a solution we can use here.